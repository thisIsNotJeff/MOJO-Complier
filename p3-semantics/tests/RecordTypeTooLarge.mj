type R = ( a,b,c,d,e,f: [16_FFF][16_FFFF]int; );
