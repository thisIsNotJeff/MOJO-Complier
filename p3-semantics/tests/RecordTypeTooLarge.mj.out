BEGIN _BUILTIN
_BUILTIN.int=int
_BUILTIN.bool=bool
_BUILTIN.false:bool=false
_BUILTIN.true:bool=true
_BUILTIN.Null=^null
_BUILTIN.nil:Null=0
_BUILTIN.Refany=^null
_BUILTIN.Address=^null
_BUILTIN.Root=null
object {
}
_BUILTIN.Text=^null
_BUILTIN.First:(...)
_BUILTIN.Last:(...)
_BUILTIN.New:(...)
_BUILTIN.Number:(...)
BEGIN RecordTypeTooLarge
RecordTypeTooLarge.R=
struct {
0: a:[4095][65535]int
1073463300: b:[4095][65535]int
2146926600: c:[4095][65535]int
-1074577396: d:[4095][65535]int
-1114096: e:[4095][65535]int
1072349204: f:[4095][65535]int
}
END RecordTypeTooLarge
END _BUILTIN
